core/deployment/pom.xml

```diff
@@ -15,6 +15,18 @@
     <name>Quarkus - Core - Deployment</name>
 
     <dependencies>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-platform-properties</artifactId>
+            <type>pom</type>
+            <version>${project.version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>*</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
```

Comment:
This could probably be removed. I added it to make sure the build order is always correct even when it's concurrent.